charter.md

```diff
@@ -53,15 +53,130 @@ The following responsibilities belong to the Steering Committe, but may be deleg
 * Control and delegate access to and establish processes regarding
   project resources/assets<sup>[6](#footnote6)</sup>  
 
-## Quorum
+## Changes
 
-Steering committee members are generally expected to attend every meeting. We
+In instances where a process is not already specified within this document,
+changes to the Steering Committee charter will be considered according to the
+processes set forth in the committee's [operations documentation][changes].
+
+[changes]: /operations/changes.md
+
+## Membership
+
+### Elections
+
+Every year, the Steering Committee holds a general election for open seats with
+[SIG Contributor Experience][sig-contribex] owning the process.
+
+SIG Contributor Experience may delegate this responsibility to a subproject.
+
+Our [election policy document][general-elections] covers the details for how
+this works.
+
+Policy and operations of special elections initiated by the Steering Committee
+for other reasons (as yet to be determined) may be handled by some agreed-upon
+[process][changes] by the committee.
+
+[general-elections]: /elections.md
+[sig-contribex]: https://git.k8s.io/community/sig-contributor-experience
+
+### Vacancies
+
+In the event of a resignation, removal, or other loss of an elected Steering 
+Committee member, the candidate with the next most votes from the previous 
+election will be offered the seat. This process will continue until the seat is 
```

Comment:
i think this may be made a bit clearer perhaps?

`If the person offered the seat declines, we repeat this process until the seat is filled`

Reply:
I continue to think that a maximum of 1 board seat should be filled this way between elections. The further down the list you go, the less likely it is that the majority of voters actually wants that person.

Reply:
> The further down the list you go, the less likely it is that the majority of voters actually wants that person.

Great point. We'll consider what could be a more complete way to handle this.

Reply:
There is some more discussion of this buried in comment on an old revision, but the other consideration is that you can leave the seat empty if it's not too long until the next regular election, and doing that is probably the best choice if e.g. the next election is less than a month away.

Reply:
Unsurprisingly, I agree with Daniel on this topic.  The list of candidates here is self-limiting - many people don;t run because they have colleagues who are running and the max-rep rules make it an XOR for them.  As such the set of people who ran "last time" does not necessarily represent the set of people who might be most appropriate if an election were held anew.

Reply:
This is not a new rule; it's in the existing election rules.  We might want to change it later, but maybe hold off on that?

Reply:
It's not a new rule but it interacts poorly with new rules, so it should probably be changed now.

Reply:
I'd planned to have the new Elections Subproject* take up this particular bit, which has needed improvement for some time, but we need an alternate procedure which we'd need to hash out.

* https://github.com/kubernetes/community/pull/6720

Reply:
If we are not waiting:

+1 on limiting replacement-by-next-most-preferred to one per year, and then going to a special election, unless the next general election is less than three months away.